packages/remix-node/fetch.ts

```diff
@@ -26,13 +22,6 @@ type NodeRequestInit = Omit<
     | Readable;
```

Comment:
It looks like we've got a type issue coming from the `Readable` here:

```
packages/remix-node/fetch.ts(59,25): error TS2345: Argument of type 'NodeRequestInit' is not assignable to parameter of type '(RequestInit & RequestExtraOptions) | undefined'.
  Type 'NodeRequestInit' is not assignable to type 'RequestInit & RequestExtraOptions'.
    Type 'NodeRequestInit' is not assignable to type 'RequestInit'.
      Types of property 'body' are incompatible.
        Type 'BodyInit | Readable | null | undefined' is not assignable to type 'BodyInit | null | undefined'.
          Type 'Readable' is not assignable to type 'BodyInit | null | undefined'.
```

That came from https://github.com/remix-run/remix/pull/2736 which states _"allow for ReadableStream's to be used as a body in node"_ so it sounds like it's relevant at first glance.

But, if we trace down into `WebRequest -> RequestInit -> BodyInit` we get:

```ts
type BodyInit = ReadableStream | XMLHttpRequestBodyInit;
```

So maybe we can remove the `Readable` and just use the underlying type since it contains `ReadableStream` already?  That would let us remove the body override entirely and simplify down to:

```ts
type NodeRequestInit = NonNullable<
  ConstructorParameters<typeof WebRequest>[1]
>;
```

If I make this change I get no type issues on code such as:

```ts
let tempInit: NodeRequestInit = { body: new ReadableStream() };
return webFetch(info, tempInit);
```